test/sanity/validate-modules/utils.py

```diff
@@ -57,8 +56,6 @@ def __enter__(self):
         self.sys_stderr = sys.stderr
         sys.stdout = self.stdout = StringIO()
         sys.stderr = self.stderr = StringIO()
-        setattr(sys.stdout, 'encoding', self.sys_stdout.encoding)
-        setattr(sys.stderr, 'encoding', self.sys_stderr.encoding)
```

Comment:
The connection, is that when `DOCUMENTATION` is invalid, `get_docstring` will call `display.error` and `display.display`.  I don't 100% remember the cause, but when writing to those fake `.stdout` and `.stderr`, an error would be raised when they did not contain a `.encoding` attribute.